src/stream/stream_backend.rs

```diff
@@ -123,10 +123,6 @@ fn check_scheme(
                 }
             }
             "udp" => {
-                if VideoEncodeType::H264 != encode {
```

Comment:
There is no explanation on the commit body or in the title explaining why this is not necessary.